api/src/main/java/org/pmiops/workbench/api/OfflineRuntimeController.java

```diff
@@ -292,6 +298,38 @@ public ResponseEntity<Void> checkPersistentDisks() {
   private boolean notifyForUnusedDisk(LeonardoListPersistentDiskResponse disk, int daysUnused)
       throws MessagingException {
     String googleProject = leonardoMapper.toGoogleProject(disk.getCloudContext());
+    StringBuilder status = new StringBuilder();
+    try {
+      // If Label contains key aou-app-type it is either cromwell or Rstudio else its Jupyter
+      // runtime
+      if (((Map) disk.getLabels()).containsKey(LEONARDO_LABEL_APP_TYPE)) {
+        List<UserAppEnvironment> appEnviornments =
+            leonardoApiClient.listAppsInProjectAsService(googleProject);
+        if (appEnviornments != null && appEnviornments.size() > 0) {
```

Comment:
If a collection can ever be a null instead of empty, that sounds problemtic and we should fix the issue inside that collection. Checked `listApp` and `listRuntime`, I don't think it would return null. So it's safe to remove null check here.